### PR TITLE
Skip negative test if no non-sctp nodes are available.

### PR DESCRIPTION
**Pull request by fedepaol**

### CODE REVIEW

File: functests/sctp/sctp.go [hunk 1]
```diff
@@ -65,6 +65,10 @@ var _ = Describe("sctp", func() {
 					LabelSelector: "node-role.kubernetes.io/worker,!" + strings.Replace(sctpNodeSelector, "=", "", -1),
 				})
 				Expect(err).ToNot(HaveOccurred())
+				if len(nodes.Items) == 0 {
```
Comment (slintes) [inline functests/sctp/sctp.go]: you can remove line 72 with this

**Comment by fedepaol** [comment]
> this is just a thought. i'm unsure if it makes sense or not.
> 
> should our expectation around having non cnf feature enabled workers present be something we feed into the functests at execution time as an argument?
> 
> reason being, it means we declare what tests should and shouldn't run rather than detect them based on the cluster's configuration.

Ah, good point. You mean, trying to avoid skipping when we shouldn't.
Something like passing the # of non cnf-workers from outiside would make the trick.

**Pull request merged**